setup-gcloud/action.yml

```diff
@@ -17,9 +16,29 @@ inputs:
       Google Cloud CLIs and APIs will automatically use this environment variable to find credentials.
     required: false
     default: 'false'
-outputs:
-  project-id:
-    description: The default project ID associated with the service account.
+
 runs:
-  using: node16
-  main: dist/index.js
+  using: composite
+  steps:
+    - name: Get gcloud latest version
```

Comment:
Is this necessary? Is this not downloaded by the setup-gcloud-base?

Won't this always download the latest version, even if it already was cached and doesn't match the version to install?

Reply:
We will, but I don't want to store the cache called "latest" because it will never install a new version otherwise, so the cache stored in the repo will be called, for example, `linux-cache-gcloud-423.0` and it will use it until the new version is released and the name would be `linux-cache-gcloud-424.0`